src/siddhiCoreJava/WeekdaySwitch.java

```diff
@@ -0,0 +1,41 @@
+package siddhiCoreJava;
+
+import java.util.Scanner;
+
+public class WeekdaySwitch {
+
+	public static void main(String[] args) 
+	
+	{
```

Comment:
what if i wanted know day on 23rd date
how you will handle this condition
think on user given date and 7

some reminder value think..